### ISSUE TITLE
If more than 75 tabs are opened, tabs added are hidden in the tab bar in the horizontal tab view

### ISSUE THREAD

**Issue by MadhaviSeelam** [brave/brave-browser#35785]
It appears that 75 open tabs are shown in the tab but any tabs after that are hidden.  They can be seen and be opened 
via tab search only.   Also more tabs are hidden seem to hidden in smaller view ports.  Verified in `1.61.101` & Verified `1.62.156` both show same. 
 
## Steps to Reproduce

   1. Install 1.62.156
   2. launch Brave
   3. open more than 75 tabs
   

## Actual result:

- Some of the tabs are hidden in the horizontal tab view
- All of the open tabs are shown in the vertical tab view 

example | example| smaller view port|vertical tab view
----- | ------ | ------- | -------
<img width="1159" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/53971ec8-7a86-4d8c-978a-07e30ada1b9c">|<img width="1159" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/22452c3b-a169-4459-b302-8de5803b3337">|<img width="1129" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/91a68abc-8825-429b-953b-bff40fa10cde">|<img width="459" alt="image" src="https://github.com/brave/brave-browser/assets/98358127/95a30585-c172-43fe-97b9-72781de51c64">

## Expected result:
Tabs should be shown?

## Reproduces how often: 

Easily 

## Brave version (brave://version info)

```
Brave | 1.62.156 Chromium: 121.0.6167.139 (Official Build) (64-bit)
-- | --
Revision | 800674fc2c6162087525ed9b5bfc07230296b27d
OS | Windows 11 Version 23H2 (Build 22631.3085)
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

@rebron

**Comment by aguscruiz** [comment]
I can't seem to open any of the screenshots, is it just me?

**Comment by stephendonner** [comment]
> I can't seem to open any of the screenshots, is it just me?

You need to be logged-in with a cookied session, to view them.

**Comment by MadhaviSeelam** [comment]
> I can't seem to open any of the screenshots, is it just me?
@aguscruiz here is the recording if you still can't view screenshots.  This was done in 1.61.x but initially I would in 1.63.x, so verified both in 1.62.x and 1.61.x)

https://github.com/brave/brave-browser/assets/98358127/01bd8951-598b-49ae-a6c5-4df05ee78d16

**Comment by aguscruiz** [comment]
That gave me anxiety 😂.
@MadhaviSeelam do you know if this is a chromium thing too?

I can think of a solution, that would be to allow scrolling (either via mouse wheel [up/down to become left/right] or trackpad. That could help solve the issue in a quick way.

Tab search honestly if you're at this point of open tabs, would be the best solution for the users though. I can't imagine someone recognizing potentially 15 identical favicons. 

@rebron what do you think about scrolling within the tab bar once the viewport becomes saturated?

**Comment by MadhaviSeelam** [comment]
> That gave me anxiety 😂. @MadhaviSeelam do you know if this is a chromium thing too?
> 
> I can think of a solution, that would be to allow scrolling (either via mouse wheel [up/down to become left/right] or trackpad. That could help solve the issue in a quick way.
> 
> Tab search honestly if you're at this point of open tabs, would be the best solution for the users though. I can't imagine someone recognizing potentially 15 identical favicons.
> 
> @rebron what do you think about scrolling within the tab bar once the viewport becomes saturated?

@aguscruiz  seeing similar behavior in Chrome. Also I am sure you have noticed, there is a larger space than normal between a open tab and new tab icon `+`, once saturation point hits the tab bar.
![image](https://github.com/brave/brave-browser/assets/98358127/3d44ba12-f355-40c6-8602-b7f2aebe64af)

**Comment by aguscruiz** [comment]
Yeah, I saw! On MacOS that space is a bit narrower. Might be a Windows only thing. But for sure that could be a bit more tight. Not that it'll make a big change once you reach that many tabs

<img width="148" alt="image" src="https://github.com/brave/brave-browser/assets/15734630/e87bb763-a4b6-4799-916d-07d754775463">
